depthai_datatype_msgs/msg/TensorInfo.msg

```diff
@@ -0,0 +1,7 @@
+int32 order
+int32 dataType
```

Comment:
Why not `TensorInfoDataType` instead of int32?

Reply:
`TensorInfoDataType` is enum.